docs/reference/api/usb/USBCDC.md

```diff
@@ -0,0 +1,32 @@
+## USBCDC
+
+<span class="images">![](https://os.mbed.com/docs/v5.9/feature-hal-spec-usb-device-doxy/class_u_s_b_c_d_c.png)<span>USBCDC class hierarchy</span></span>
+
+The USBCDC class emulates a basic serial port over USB. You can use this serial port to send or receive data. If you want printf functionality, please see the [USBSerial](USBSerial.html) class.
```

Comment:
What are some best practices for using USBCDC? What are some real-world situations when you'd use it? What are some concepts you need to understand to use it?